test/e2e/bundle_e2e_test.go

```diff
@@ -47,6 +118,30 @@ var _ = Describe("Installing bundles with new object types", func() {
 				imageName  = "quay.io/olmtest/single-bundle-index:pdb"
 			)
 
+			// create VPA CRD on cluster
+			data, err := yaml.YAMLToJSON([]byte(vpaCRDManifest))
+			Expect(err).ToNot(HaveOccurred(), "could not convert vpa crd to json")
+
+			crd := &apiextensionsv1beta1.CustomResourceDefinition{}
+			err = json.Unmarshal(data, crd)
+			Expect(err).ToNot(HaveOccurred(), "could not convert vpa crd to unstructured")
```

Comment:
You're unmarshaling it to a CRD, not an unstructured -- though it's probably better to use an unstructured to make bumping the fixture version easier.